### PR TITLE
Feature/synapse io website

### PR DESCRIPTION
**Pull request by guttff**
Creating the synapse io website

## Description
Creating the synapse io website available at https://americanexpress.io/synapse/ 
This site contains our mission statement as well as the team, and description of what is synapse and how to onboard.

## Changes in this pull request
the images
the stylesheets
the fonts
the javascript and parallax logic

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -0,0 +1,445 @@
...
+                    </div>
+                </div>
+                <div class="col-md-3 col-sm-6 col-xs-12">
+                    <div class="service-item">
+                        <div class="icon">
+                            <img src="img/service_icon_04.png" alt="">
+                        </div>
+                        <h4>High Quality Built-In</h4>
+                        <div class="line-dec"></div>
+                        <p>Covers a majority of the scenarios you will need when creating an enterprise-grade application, it is also very open to extension.</p>
```
Comment (pclaudio3) [inline docs/index.html:126]: I would either change the comma splice from "application, it is" to "application. It is " or just delete the "it is also very open to extension" part

File: docs/index.html [hunk 2]
```diff
@@ -0,0 +1,445 @@
...
+        <div class="content-wrapper">
+            <div class="inner-container container">
+                <div class="row">
+                    <div class="col-md-4 col-sm-12">
+                        <div class="section-heading">
+                            <h4>Getting Started</h4>
+                            <div class="line-dec"></div>
+                            <p>Categories</p>
+                            <div class="filter-categories">
+                                <ul class="project-filter">
```
Comment (pclaudio3) [inline docs/index.html:146]: Can this text be larger?

File: docs/index.html [hunk 3]
```diff
@@ -0,0 +1,445 @@
...
+                        <div class="item">
+                            <div class="testimonials-item">
+                                <p>“ Quisque ullamcorper odio a nisl lacinia dictum. Vestibulum malesuada ipsum in turpis finibus, ut sagittis erat scelerisque. Curabitur non risus fringilla libero accumsan molestie et quis justo. ”</p>
+                                <h4>George Rich</h4>
+                                <span>Marketing Head</span>
+                            </div>
+                        </div>
+                        <div class="item">
+                            <div class="testimonials-item">
+                                <p>“ Suspendisse magna mauris, convallis vel finibus eget, lobortis dictum neque. Nam tincidunt metus nec eros dignissim consectetur. ”</p>
```
Comment (pclaudio3) [inline docs/index.html:226]: All of this should change right

File: docs/index.html [hunk 4]
```diff
@@ -0,0 +1,445 @@
...
+
+    <div class="tabs-content" id="blog">
+        <div class="container">
+            <div class="row">
+                <div class="wrapper">
+                    <div class="col-md-4">
+                        <div class="section-heading">
+                            <h4>Support Documentation</h4>
+                            <div class="line-dec"></div>
+                            <p>Find modules here, for reference and a quick start.</p>
```
Comment (pclaudio3) [inline docs/index.html:259]: Can you remove the comma please

File: docs/index.html [hunk 5]
```diff
@@ -0,0 +1,445 @@
...
+                        </div>
+                    </div>
+                    <div class="col-md-8">
+                        <section id="first-tab-group" class="tabgroup">
+                            <div id="tab1">
+                                <img src="img/blog_item_01.jpg" alt="">
+                                <div class="text-content">
+                                  <h4>Service Rest</h4>
+                                    <span><a href="#">Synapse</a> / <a href="#">21 September 2021</a></span>
+                                  <p>This is the synapse gateway framework utilized to consume RESTful APIs. It provides several out-of-the-box functionalities.</p>
```
Comment (pclaudio3) [inline docs/index.html:275]: Can you uppercase the "S" in "Synapse"

File: docs/index.html [hunk 6]
```diff
@@ -0,0 +1,445 @@
...
+                                    <span><a href="#">Synapse</a> /  <a href="#">24 August 2020</a></span>
+                                    <p>This is the synapse gateway framework utilized to consume RESTful APIs. It provides several out-of-the-box functionalities</p>
+                                </div>
+                            </div>
+                            <div id="tab3">
+                                <img src="img/blog_item_03.jpg" alt="">
+                                <div class="text-content">
+                                    <h4>Couchbase</h4>
+                                    <span><a href="#">Synapse</a> /  <a href="#">18 July 2020</a></span>
+                                    <p>This is the synapse couchbase abstraction framework used whenever there is a need to connect to couchbase database and read from it. It provides several out-of-the-box functionalities. </p>
```
Comment (pclaudio3) [inline docs/index.html:291]: Can you uppercase all the S in Synapse and all the C in Couchbase

File: docs/index.html [hunk 7]
```diff
@@ -0,0 +1,445 @@
...
+    </div>
+    <!--/.header-->
+
+
+    <div class="parallax-content baner-content" id="home">
+        <div class="container">
+            <div class="text-content">
+<h1><em><img src="img/synapse_logo-reversed-white.png" alt=""/>&nbsp;</em> <span>&nbsp;</span> </h1>
+				<h2><strong>The future of development is here.</strong></h2>
+              <p>A Library of documentation code built-in just for you!</p>
```
[the remark this answers is not in the document]
Comment (guttff) [reply docs/index.html:68]: done

File: docs/index.html [hunk 8]
```diff
@@ -0,0 +1,445 @@
...
+                    </div>
+                </div>
+                <div class="col-md-3 col-sm-6 col-xs-12">
+                    <div class="service-item">
+                        <div class="icon">
+                            <img src="img/service_icon_02.png" alt="">
+                        </div>
+                        <h4>Couchbase Library</h4>
+                        <div class="line-dec"></div>
+                        <p>Provides a couchbase library to build dynamic queries based on Spring Data and Query DSL</p>
```
[the remark this answers is not in the document]
Comment (guttff) [reply docs/index.html:100]: done

File: docs/index.html [hunk 9]
```diff
@@ -0,0 +1,445 @@
...
+
+    <section id="about" class="page-section">
+        <div class="container">
+            <div class="row">
+                <div class="col-md-3 col-sm-6 col-xs-12">
+                    <div class="service-item">
+                        <div class="icon">
+                            <img src="img/service_icon_01.png" alt="">
+                        </div>
+                        <h4>Simple and Lightweght</h4>
```
[the remark this answers is not in the document]
Comment (guttff) [reply docs/index.html:85]: done

File: docs/index.html (hunk 1 above)
Comment (guttff) [reply docs/index.html:126]: done

File: docs/index.html (hunk 2 above)
Comment (guttff) [reply docs/index.html:146]: done

File: docs/index.html [hunk 10]
```diff
@@ -0,0 +1,445 @@
...
+                                    <span><a href="#">Synapse</a> / <a href="#">21 September 2021</a></span>
+                                  <p>This is the synapse gateway framework utilized to consume RESTful APIs. It provides several out-of-the-box functionalities.</p>
+                                </div>
+                            </div>
+                            <div id="tab2">
+                                <img src="img/blog_item_02.jpg" alt="">
+                                <div class="text-content">
+                                    <h4>Client Rest</h4>
+                                    <span><a href="#">Synapse</a> /  <a href="#">24 August 2020</a></span>
+                                    <p>This is the synapse gateway framework utilized to consume RESTful APIs. It provides several out-of-the-box functionalities</p>
```
[the remark this answers is not in the document]
Comment (guttff) [reply docs/index.html:283]: done

File: docs/index.html (hunk 6 above)
Comment (guttff) [reply docs/index.html:291]: done

**Pull request closed without merging**